### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by saingsab** [bitriel/bitriel-wallet#4] [General]
Hi Welcome @kruyvanna 
Please feel free to submit any issue.

**Comment by kruyvanna** [comment]
Thank you saing for coordinating and hospitality.
I enjoyed this trip

**Reply by rithythul** [reply]
I am quite late for the party :) welcome @kruyvanna

**Reply by kruyvanna** [reply]
Thanks Rithy 🌱
